src/engine/stream_manager.h

```diff
@@ -117,6 +120,9 @@ RunContext StreamManager<kNumGpus, kStreams>::GetIORunContext(
       LOG(FATAL) << MXNET_GPU_NOT_ENABLED_ERROR;
```

Comment:
Try adding a `break` here and see what VC++ says?

Reply:
Thanks, but this doesn't result in any change. It still complains about multiple default cases